### ISSUE TITLE
Resize up feature

### ISSUE THREAD

**Issue by kawasaki656** [vkurko/calendar#343]
There is only resize bottom feature, do you have chance to implement resize top also ? It would be helpfull, thanks

**Comment by vkurko** [comment]
You are right. I will keep this ticket open to see how popular this request is.

**Comment by TimNZ** [comment]
+1

**Comment by Retrovers** [comment]
+ 1

**Comment by Porowl** [comment]
+1

**Comment by volpeo** [comment]
+1

**Comment by kawasaki656** [comment]
@vkurko thanks in advance for supporting the library

**Comment by digidoed** [comment]
+1 vkurko, you are great! Can I buy you a .... drink?

**Comment by vkurko** [comment]
@digidoed You can become a [sponsor](https://github.com/sponsors/vkurko) of the project. Thank you in advance.

**Comment by vkurko** [comment]
> Any updates on this feature?

This feature is not planned yet. I have added a one-time payment option for this feature request on the [sponsors](https://github.com/sponsors/vkurko?frequency=one-time) page. If anyone wants to donate part of the amount, they can change the amount to a smaller one in the input field.

**Comment by vkurko** [comment]
> I paid the total amount on the 19/12, do you know it will be available please ?

Sorry, I'm on vacation now. I'll get to work on this task as soon as I'm back in January.

**Comment by vkurko** [comment]
@louismathe The _Resize up feature_ should be available in v3.8.0. Please see [eventResizableFromStart](https://github.com/vkurko/calendar?tab=readme-ov-file#eventresizablefromstart) option. Thank you for sponsoring the project!

**Comment by vkurko** [comment]
Great!
